enos/README.md

```diff
@@ -165,9 +165,8 @@ state will be managed by Terraform Cloud. Here are the steps to configure the Gi
 - Access to the `hashicorp-qti` organization in Terraform Cloud.
 - Full access to the CI AWS account is required.
 
-**Note:** For help with access to Terraform Cloud and the CI Account, contact the QT team on
-the Slack channel mentioned above. Access to the AWS account can be done via Doormat, at: 
-https://doormat.hashicorp.services/.
+**Note:** For help with access to Terraform Cloud and the CI Account, contact the QT team on Slack 
+(#team-quality). Access to the AWS account can be done via Doormat, at: https://doormat.hashicorp.services/.
```

Comment:
Good stuff. Perhaps we should also include that they need to request access for the `vault_ci` account via Doormat. That is done in via Cloud Access > AWS > Request Account Access.

The idea being that we tell them exactly what to do and they don't have to try and figure out what to do in Doormat.

Reply:
Done.